gr00t/experiment/trainer.py

```diff
@@ -29,6 +30,10 @@
     is_sagemaker_mp_enabled,
 )
 
+torch.serialization.add_safe_globals(
```

Comment:
Thanks for the fix, very helpful! Can we put this in the `class DualBrainTrainer` init. and add a short comment.